src/ray/common/cgroup/cgroup_setup.h

```diff
@@ -23,8 +23,9 @@ namespace internal {
 // Return whether current user could write to cgroupv2.
 bool CanCurrenUserWriteCgroupV2();
 
-// Return whether cgroup V2 is mounted in read and write mode.
-bool IsCgroupV2MountedAsRw();
+// Return whether cgroup V2 is mounted in read and write mode exclusively on the given
+// [path] (with no cgroupv1 meanwhile).
```

Comment:
pls follow the existing standard format for args & return values.

e.g., https://github.com/ray-project/ray/blob/2bd1a560cb4317501ab23d329887b026a92047c1/src/ray/core_worker/core_worker.h#L270

Reply:
pls also make this more precise. ex:
```suggestion
// Return whether cgroup V2 is mounted in read and write mode exclusively on the given path. Also checks that cgroupv1 is not mounted.
```